README.md

```diff
@@ -45,7 +45,7 @@ In addition to this README, the following is available in the [flutter plugin Gi
 * Xcode version: 10 or greater.
 * iOS Base SDK: 12 or greater.
 * Deployment target: iOS 12.0 or greater.
-* Currently, this plugin will work only on Mac with Intel processor. This will not work on Mac with M1 chip.
+* This plugin will work on Mac with Intel processor and Apple sillicon chips (M1 and M2).
```

Comment:
I'd omit the parenthetical---if we had problems with M2 but not M1, we'd have to say that.  If we don't, "Apple silicon chips" is sufficient.

Also, one L in silicon.